charmstore/client.go

```diff
@@ -85,6 +85,7 @@ func makeWrapper(bakeryClient *httpbakery.Client, server *url.URL) csWrapper {
 	p := csclient.Params{
 		BakeryClient: bakeryClient,
 	}
+	// veebers: I don't think this would be nil anymore as we're being specific.
```

Comment:
A code grep shows this is called by charmrevisionupdater which passes in nil.
So that needs to be fixed.
If we expect the url to not be nil, we need to check that and return an error if it is.